pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -206,14 +207,14 @@ func (ca *cloudCIDRAllocator) retryParams(nodeName string) (bool, time.Duration)
 }
 
 func nodeUpdateRetryTimeout(count int) time.Duration {
-	timeout := updateRetryTimeout
-	for i := 0; i < count && timeout < maxUpdateRetryTimeout; i++ {
+	timeout := updateRetryTimeoutMilliseconds
+	for i := 0; i < count && timeout < maxUpdateRetryTimeoutMilliseconds; i++ {
 		timeout *= 2
 	}
-	if timeout > maxUpdateRetryTimeout {
-		return maxUpdateRetryTimeout
+	if timeout > maxUpdateRetryTimeoutMilliseconds {
+		timeout = maxUpdateRetryTimeoutMilliseconds
```

Comment:
And then you here would need:
```
time.Duration(rand.Int31n(timeout/time.Millisecond)) * time.Millisecond
```
But I still think it's cleaner.

Reply:
I agree. Changed it.